test/testenv/verificationutils.go

```diff
@@ -569,3 +569,38 @@ func VerifyAppFrameworkDeployment(standalone *enterprisev1.Standalone) {
 	}
 
 }
+
+// VerifyAppInstall verify that app of specific version is installed
+func VerifyAppInstall(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps map[string]string, versionCheck bool, statusCheck string) {
+	for appName, version := range apps {
+		status, versionInstalled, err := GetPodAppStatus(deployment, podName, ns, appName)
+		logf.Log.Info("App info returned for app", "App-name", appName, "status", status, "versionInstalled", versionInstalled)
+		gomega.Expect(err).To(gomega.Succeed(), "Unable to get app status on pod ")
+		gomega.Expect(strings.ToLower(status)).Should(gomega.Equal(strings.ToLower(statusCheck)))
+		if versionCheck {
+			gomega.Expect(versionInstalled).Should(gomega.Equal(version))
+		}
+	}
+}
+
+// VerifyAppsCopied verify that apps are copied to correct location based on POD
+func VerifyAppsCopied(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps []string) {
+	path := "etc/apps"
+	if strings.Contains(podName, "cluster-master") {
+		path = "etc/master-apps/_cluster"
+	}
+	if strings.Contains(podName, "-deployer-") {
+		path = "etc/shcluster/apps"
+	}
+	VerifyAppsInFolder(deployment, testenvInstance, ns, podName, apps, path)
+}
+
+// VerifyAppsInFolder verify that apps are present in folder
+func VerifyAppsInFolder(deployment *Deployment, testenvInstance *TestEnv, ns string, podName string, apps []string, path string) {
+	appList, err := GetSubDirsOnPod(deployment, podName, path)
+	gomega.Expect(err).To(gomega.Succeed(), "Unable to get apps on pod", "Pod", podName)
+	for _, app := range apps {
+		found := CheckStringInSlice(appList, app)
+		gomega.Expect(found).Should(gomega.Equal(true))
```

Comment:
Nit: Add a log line here. It will be really hard to debug the failures without any logging.